implementations/elixir/ockam/ockam_cloud_node/config/runtime.exs

```diff
@@ -110,13 +94,11 @@ config :ockam, identity_module: identity_module
 
 ## Services config
 
-services_json = System.get_env("SERVICES_JSON")
-
-services_list = System.get_env("SERVICES_LIST")
+services_list = System.get_env("SERVICES_LIST", "")
 
-services_file = System.get_env("SERVICES_FILE")
-
-services_config_source = System.get_env("SERVICES_CONFIG_SOURCE")
+services =
+  String.split(services_list, ",", trim: true)
+  |> Enum.map(fn name -> String.trim(name) |> String.to_atom() end)
```

Comment:
This is somewhat of a change in behavior as the application will no longer start any services by default, and in order to start services on startup a specifically formatted services list has to be provided as an env variable.

We don't really have any readme where we could make note about this, but I just wanted to mention that this change in behavior and the way to set up services for `ockam_cloud_node` is currently not very visible.

Reply:
Yes. This is a breaking change on purpose. We don't currently document usage of `ockam_cloud_node` currently and it's only used in some internal components.